src/components/Article/ArticleController.tsx

```diff
@@ -275,28 +276,35 @@ const ArticleController = (props: Props) => {
 
       <OakViewResolver sideLabel="More options">
         <OakView main>
+          <OakButton
+            action={() => setEditDialogOpen(!editDialogOpen)}
+            theme="primary"
+            variant="animate out"
+            align="right"
+          >
+            <i className="material-icons">double_arrow</i>New Request
+          </OakButton>
+          <div className="search-bar">
+            <div />
+            <div>
+              <OakText
+                label="Search"
+                data={data}
+                handleChange={find}
+                id="searchCriteria"
+              />
+            </div>
+            <div className="clear">
+              <OakButton theme="default" variant="block" icon="clear" />
```

Comment:
add an inline function here handleChange = {() => setSearchCriteria('')}

Reply:
Changes done as per the comments